plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/status/StatusUtilTest.java

```diff
@@ -99,14 +98,14 @@ public void testError_withClassAndException() {
 
   @Test
   public void testError_withInstance() {
-    IStatus error = StatusUtil.error(new DeleteAllVisitor(), "test error msg");
+    IStatus error = StatusUtil.error("an object", "test error msg");
```

Comment:
I think you can just use `this`: the test class is hosted by bundle.

Reply:
This one and the below are still `"an object"`.